### DISCUSSION TITLE
I am getting Dependabot alerts  as PostCSS line return parsing error , How can i resolve this?

### DISCUSSION THREAD

**Discussion by mallikarjuna11421** [postcss/postcss#1891] [Q&A]
Hi ,

I am getting   PostCSS line return parsing error as Dependabot alerts.

I am using react-scripts@5.0.1 , can you help me to resolve this issue. you can find the screenshot in the below

![image](https://github.com/postcss/postcss/assets/92287239/2b36691d-2713-4e18-a762-d94146254ce5)

**Comment by ai** [comment]
You need así `react-script` team to update their plugin to PostCSS 8.

7.x branch is a few years old and is not supported anymore. We have less resources that React team.

**Reply by mallikarjuna11421** [reply]
Okay Thanks @ai
